packages/flutter_tools/lib/src/android/android_workflow.dart

```diff
@@ -31,7 +31,7 @@ final RegExp licenseNotAccepted = new RegExp(r'licenses? not accepted', caseSens
 final RegExp licenseAccepted = new RegExp(r'All SDK package licenses accepted.');
 
 class AndroidWorkflow extends DoctorValidator implements Workflow {
-  AndroidWorkflow() : super('Android toolchain - develop for Android devices');
```

Comment:
Why do you want to remove the part after the hyphen here and below?

Reply:
it was referenced/commented about in the issue https://github.com/flutter/flutter/issues/6304#issuecomment-283058990

I could remove If you'd like

Reply:
Seems reasonable, no need to roll back.